addons/kafka/tests/kafka-install/00-install.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl apply -f ./../../../zookeeper/0.x/zookeeper-3.yaml
```

Comment:
It looks like currently we're going to have to remember to manually change these every time we add new revisions. I would suggest we do something about this now if possible, OR create a JIRA documenting this so that we can plan some improvements (my first thought is that a catalog CLI which can simply feed you the latest revision would be nice)

Reply:
let me check if there is a nice way of automating this

Reply:
let's create a JIRA as I have talked with @kensipe also and we might work in direction of Suite definition in KUTTL providing this kind of resources which can we later automate also